src/bookshelf/rebar.config

```diff
@@ -31,7 +31,7 @@
 
 {erl_opts, [debug_info,
             {parse_transform, lager_transform},
-            warnings_as_errors}].
+            warnings_as_errors]}.
```

Comment:
@stevendanna is right. rebar.config.lock was giving us a pass on this. Although I have no idea how the last person to generate it was successful